pennylane_ionq/device.py

```diff
@@ -60,6 +73,18 @@
 }
 
 
+class CircuitIndexNotSetException(Exception):
+    """Raised when after submitting multiple circuits circuit index is not set
+    before the user want to access implementation methods of IonQDevice
+    like probability(), estimate_probability(), sample() or the prob property.
+    """
+
+    def __init__(self):
+        self.message = "Because multiple circuits have been submitted in this job, the index of the circuit \
+you want to acces must be first set via the set_current_circuit_index device method."

```

Comment:
```suggestion
        self.message = (
            "Because multiple circuits have been submitted in this job, the index of the circuit "
            "you want to access must be first set via the set_current_circuit_index device method."
        )

```

Reply:
fixed